samples/RoutingSample.Web/UseEndpointRoutingStartup.cs

```diff
@@ -90,11 +99,9 @@ public void Configure(IApplicationBuilder app)
                         }
 
                         return Task.CompletedTask;
-                    },
-                    "/graph",
-                    "DFA Graph",
-                    new HttpMethodMetadata(new[] { "GET", }));
-                builder.MapEndpoint(
+                    });
+                builder.MapGet(
+                    "/WithSingleAsteriskCatchAll/{*path}",
```

Comment:
What would you think about trimming down the set of things that are in this class. They aren't super meaningful, and I'm OK with just deleting the functional tests that cover them.

Reply:
I'd like to keep them. These functional tests helped me spot a bug yesterday.

What about splitting this site into a site for functional tests and a sandpit for experiments?

Reply:
Splitting the site can wait for another PR

Reply:
Yeah that sounds good to me. I'd really prefer that the sample evolve based on what we're working on, or what's important for users to see. That requires us to not try and keep stuff just because it was valuable at some point 👍 functional test are that.